graphdatascience/query_runner/aura_db_query_runner.py

```diff
@@ -50,8 +53,15 @@ def call_procedure(
         if params is None:
             params = CallParameters()
 
-        if AuraDbQueryRunner.GDS_REMOTE_PROJECTION_PROC_NAME == endpoint:
-            return self._remote_projection(endpoint, params, yields, database, logging)
+        if AuraDbQueryRunner.GDS_REMOTE_PROJECTION_PROC_NAME in endpoint:
+            if self._endpoint_has_supported_version(endpoint):
+                return self._remote_projection(endpoint, params, yields, database, logging)
```

Comment:
so in this PR we send the same remote projection query regardless of protocol version?

Reply:
Fixed now